cmd/promtool/main.go

```diff
@@ -204,6 +204,7 @@ func main() {
 	pushMetricsHeaders := pushMetricsCmd.Flag("header", "Prometheus remote write header.").StringMap()
 
 	testCmd := app.Command("test", "Unit testing.")
+	junitOutFile := testCmd.Flag("junit", "The JUnit xml output file.").OpenFile(os.O_CREATE|os.O_WRONLY, 0o644)
```

Comment:
`The JUnit...` assumes the reader understands what this is. This will end up in docs too, so more clear would be something like: `Output test results in JUnit XML format to this file.`

`OpenFile` probably wants O_TRUNC too.